form/form_collections.rst

```diff
@@ -720,3 +739,4 @@ the relationship between the removed ``Tag`` and ``Task`` object.
 .. _`JSFiddle`: http://jsfiddle.net/847Kf/4/
 .. _`@a2lix/symfony-collection`: https://github.com/a2lix/symfony-collection
 .. _`symfony-collection`: https://github.com/ninsuo/symfony-collection
+.. _`Twig macros`: https://twig.symfony.com/doc/2.x/tags/macro.html
```

Comment:
Do twig doc provide a `current` doc? Defining a version like this is hard to maintain in the long run no?